usage-rules.md

```diff
@@ -2,7 +2,25 @@
 
 ## Understanding Ash
 
-Ash is an opinionated, composable framework for building applications in Elixir. It provides a declarative approach to modeling your domain with resources at the center. Read documentation  *before* attempting to use it's features. Do not assume that you have prior knowledge of the framework or its conventions.
+Ash is an opinionated, composable framework for building applications in Elixir. It provides a declarative approach to modeling your domain with resources at the center.
+
+Always use Ash concepts, almost never ecto concepts directly. Think hard about the "Ash way" to do things. If you don't know, often look for information in the rules & docs of Ash & associated packages.
+
+## Development Tools & Workflow
+
+### MCP Tools
+
+- Use Tidewave MCP tools when available, as they let you interrogate the running application in various useful ways.
+- The Tidewave and Ash MCP tools require the Phoenix server to be running in the background via `mix phx.server` which never terminates.
+
+### Research and Documentation
+
+Always use package_docs_search to find relevant documentation before beginning work. ALWAYS research, NEVER assume.
```

Comment:
specific to tidewave, but also we have to be careful not to install our own ethos in terms of the agents workflows into these files. That should be up to the end user.